### PR TITLE
[feature] Evenements

### PR DESCRIPTION
**Pull request by aleckvincent**
## Title:
Add `Événements` functionality to actions and timeline

## Description:
This pull request introduces the `Événements` functionality to both actions and the timeline. 

## Changes Made:
- Integrated `Événements` into the timeline
- Added `Événements` functionality to actions

## Testing:
- Tested the functionality thoroughly to ensure proper integration
- Conducted unit tests to verify the accuracy of `Événements` handling
- Reviewed and tested edge cases to ensure robustness

### CODE REVIEW

File: backend/src/main/kotlin/fr/gouv/dgampa/rapportnav/infrastructure/bff/model/action/EnvActionData.kt [hunk 1]
```diff
@@ -30,5 +30,6 @@ data class EnvActionData(
     val controlAdministrative: ControlAdministrative? = null,
     val controlGensDeMer: ControlGensDeMer? = null,
     val controlNavigation: ControlNavigation? = null,
-    val controlSecurity: ControlSecurity? = null
+    val controlSecurity: ControlSecurity? = null,
+    val coverMissionZone: Boolean? = null,
```
Comment (lwih) [inline backend/src/main/kotlin/fr/gouv/dgampa/rapportnav/infrastructure/bff/model/action/EnvActionData.kt:34]: C'est quoi ce coverMissionZone ? Ca m'évoque rien

Comment (aleckvincent) [reply backend/src/main/kotlin/fr/gouv/dgampa/rapportnav/infrastructure/bff/model/action/EnvActionData.kt:34]: Il est retourné dans la response de l'API MonitorEnv. Il n'est pas utile pour la feature "evenement" mais je l'ai rajouté pour compléter aux cas où si besoin pour les features futures.
